pkgs/applications/graphics/upscayl/default.nix

```diff
@@ -0,0 +1,38 @@
+{
+  appimageTools,
+  fetchurl,
+  lib,
+}: let
+  pname = "upscayl";
+  version = "2.5.5";
+  name = "${pname}-${version}";
+  src = fetchurl {
+    url = "https://github.com/upscayl/upscayl/releases/download/v${version}/upscayl-${version}-linux.AppImage";
+    hash = "sha256-qpLxOGphR9iHvtb8AZZaMict/g8wLkL7Dhr4mt3LZdk=";
+  };
+  appimageContents = appimageTools.extractType2 {
+    inherit name src;
+  };
+in
+  appimageTools.wrapType2 rec {
```

Comment:
```suggestion
  appimageTools.wrapType2 {
```

Reply:
Reminds me to fix my nix-mode setup.. fixed! xD